UT4MasterServer/Services/ApplicationBackgroundService.cs

```diff
@@ -51,6 +51,11 @@ private void DoWork(object? state)
 			if (deleteCount > 0)
 				logger.LogInformation("Background task deleted {DeleteCount} expired sessions.", deleteCount);
 
+			var codeService = scope.ServiceProvider.GetRequiredService<CodeService>();
+			deleteCount = await codeService.RemoveAllExpiredCodesAsync();
+			if (deleteCount > 0)
+				logger.LogInformation("Background task deleted {DeleteCount} expired codes.", deleteCount);
```

Comment:
If you are trying to save some space by adding condition on the log message, why just not remove it?
If you aren't trying to save some space, just remove IF.

Reply:
i am trying to remove clutter in console and there's no reason to say when nothing was deleted. I was at some point annoyed because it was spitting out 2 lines of practically no info in debug every minute .